Sources/DeploymentTargetAWSLambda/main.swift

```diff
@@ -285,8 +285,9 @@ struct LambdaDeploymentProvider: DeploymentProvider {
         try runInDocker(
             imageName: dockerImageName,
             bashCommand: [
-                "swift", "run", productName,
-                "--mode=" + WellKnownCLIArguments.exportWebServiceModelStructure, "--fileurl=.build/\(tmpDirName)/\(filename)" // can't use self.tmpDirUrl here since that's an absolute path but we need a relative one bc this is running in the docker container which has a different mount path
+                "\(WellKnownEnvironmentVariables.executionMode)=\(WellKnownEnvironmentVariableExecutionMode.exportWebServiceModelStructure)",
```

Comment:
same as above, this should be working fine. However, I'd prefer we instead add an `environment` parameter to the `runInDocker` function, and use docker's `--env` option to specify the container's environment variables. I'll implement that in my branch (unless of course you want to make that change)

Reply:
I'll take a closer look at that in the evening if it's not done until then :)